public/visualization_fn.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { get } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { PersistedState } from '../../../src/plugins/visualizations/public';
+import { ExpressionFunctionDefinition, Render } from '../../../src/plugins/expressions/public';
+import { getIndexPatterns, getFilterManager, getSearchService, getVisualization } from './services';
+import { enhancedTableRequestHandler } from './data_load/enhanced-table-request-handler';
+import { enhancedTableResponseHandler } from './data_load/enhanced-table-response-handler';
+import { documentTableResponseHandler } from './data_load/document-table-response-handler';
+
+import { DOC_TABLE_VIS_NAME } from './types';
+import { ENH_TABLE_VIS_NAME }  from './types';
+import { VisName } from './types';
```

Comment:
could you merge these 3 lines in only one line?

Reply:
Merged the three imports in one line